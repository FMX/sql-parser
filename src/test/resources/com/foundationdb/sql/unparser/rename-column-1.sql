RENAME COLUMN IF EXISTS test.t.c1 TO c2