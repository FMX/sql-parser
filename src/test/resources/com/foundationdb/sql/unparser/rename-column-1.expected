ALTER TABLE IF EXISTS test.t RENAME COLUMN c1 TO c2